#Section variables
string id_section
string name
string qr_docking
string maps_path
string description
string id_shop
string prev_section
